testing/postgres-client-tests/node/workbenchTests/diffs.js

```diff
@@ -248,8 +389,22 @@ export const diffTests = [
           diff_type: "schema",
           statement: "DROP TABLE `test_info`;",
         },
+        // TODO: Should `CREATE SCHEMA` statement be included here?
         {
           statement_order: "2",
+          from_commit_hash: "r6g8g61k89dpgb3cuks70jfnavr6b1q0",
+          to_commit_hash: "WORKING",
+          table_name: "testanother",
+          diff_type: "schema",
+          statement:
+            "CREATE TABLE `testanother` (\n" +
```

Comment:
These won't parse for postgres, we need custom logic for it. Maybe include this in the issue you file above